src/hooks/useFontLoader.tsx

```diff
@@ -0,0 +1,17 @@
+import { useAtom } from "jotai"
+
+import { $theme } from "../store/theme"
+
+export function FontLoader() {
+  const [theme] = useAtom($theme)
+
+  return (
+    <div>
```

Comment:
Since it renders a `link` component, I don't think it needs to be nested under `div`. Like you could just return `link` directly.

Reply:
Nice catch, I changed this file a lot so I forgot to clean it up 🙏🏻